### DISCUSSION TITLE
Repository for previews? (Specifically RP2350 / Pico 2)

### DISCUSSION THREAD

**Discussion by inst4bility** [micropython/micropython#15859] [Core Development] [answered]
Hi,

Is there a repository for the source for the preview firmwares? In particular, I'm looking for the source that built the Pico 2 preview (RPI_PICO2-20240809-v1.24.0-preview.201.g269a0e0e1.uf2), since at the moment the Pico 2 doesn't have any official releases. While I respect it's still in preview state, It appears pretty stable for my application, however I need to add some libraries to it.

Thanks in advance!

**Comment by ricksorensen** [comment, marked as answer]
Here is a pull request for the RP2350 - does this address what you are looking for?

#15619

**Reply by inst4bility** [reply]
Thanks, that'll do it.

**Reply by mattytrentini** [reply]
Also, note one of the comments in that PR: pre-built firmware binaries are available from the download site: https://micropython.org/download/RPI_PICO2/
